### ISSUE TITLE
Camera Lock (WASD) on opening

### ISSUE THREAD

**Issue by Maxwelllondon92** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#384]
**Describe the bug**
After opening a campaign, once loaded i cannot move the camera with wasd keys.
Only after using the mid-click camera movement I can then use the wasd keys.

**To Reproduce**
1. Load a Campaign
2. Wait for it to launch
3. Try to move the camera with wasd

**Comment by LuPro** [comment]
Duplicate of #130
